project-resources/README.md

```diff
@@ -73,4 +77,17 @@ and discussions as guidance when determining the content of their updates.
 
 It is highly recommended that you seek peer review for your updates beyond that
 of the Technical Leads and Chairs. More information on contributions to this
-repo may be found in the [contributing file](../CONTRIBUTING.md)
+repo may be found in the [contributing file](../CONTRIBUTING.md).
```

Comment:
actually this link is now supposedly https://github.com/cncf/tag-security/blob/main/CONTRIBUTING/README.md 
```suggestion
repo may be found in the [contributing file](../CONTRIBUTING.md).
```

Reply:
```suggestion
repo may be found in the [contributing file](../CONTRIBUTING/README.md).
```